index.html

```diff
@@ -0,0 +1,160 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="style.css">
+
+    <script src="https://kit.fontawesome.com/a03b3724cb.js" crossorigin="anonymous"></script>
+
+    <title>2021 CC global Summit</title>
+</head>
+
+<body>
```

Comment:
- [ ] Let's make sure we have only one `h1` tag on each page both the about and index for  SEO reasons and to make your code look more semantic.

Reply:
* Fixed 👍